nasa/harmony-gdal-adapter
# HGA - Update to GDAL 3.6.0.

## Description

This PR picks up on a suggestion from @flamingbear from one of the previous HGA PRs: updating the base Docker image so that:

* We are using a more up-to-date version of GDAL (3.6.0 vs 3.4.3).
* We are using more up-to-date geospatial Python depedendencies.
* We are using Python 3.10 (allowing that glorious context manager with parentheses syntax).

## Jira Issue ID

N/A

## Local Test Steps

Same as the last few PRs:

* Pull the branch locally
* Build the new image (`bin/build-image`).
* Restart the services in your local Harmony instance (`bin/restart-services` in the Harmony repository)
* Run the HGA regression test notebook. All tests should pass.

## PR Acceptance Checklist
* ~~Jira ticket acceptance criteria met.~~
* ~~Tests added/updated and passing.~~
* ~~Documentation updated (if needed).~~
